ibis/omniscidb/tests/test_client.py

```diff
@@ -310,3 +340,29 @@ def test_cpu_execution_type(
 
     for mocked_method in mocked_methods:
         mocked_method.stop()
+
+
+@pytest.mark.parametrize(
+    'method,data',
+    [
+        ('rows', df_salary),
+        ('columnar', df_salary),
+        ('infer', df_salary),
+        pytest.param(
+            'arrow',
+            arrow_salary,
+            marks=pytest.mark.skipif(
+                pyarrow is None, reason='PyArrow is not installed'
```

Comment:
is an arrow an explicit optional dependency of ominisc? can you update setup.py 
are we testing this on CI?

Reply:
I was checking the pymapd right now .. and it seems it is required now. thanks @jreback I will fix that now.

Reply:
also I opened an issue on pymapd related to pymapd.load_data docstring as now it seems pyarrow is required.

Reply:
@xmnlab I was checking this again. Can you add `pyarrow` to the `omniscidb_requires` in `setup.py` please? Since we're importing it, it's worth having the requirement explicit, even if it's a `pymapd` dependency.

Reply:
done!  thanks @datapythonista !